openmetadata-spec/src/main/antlr4/org/openmetadata/schema/EntityLink.g4

```diff
@@ -132,5 +132,5 @@ ENTITY_FIELD
     
 
 NAME_OR_FQN
-    : ~(':')+ ('>')*? ~(':'|'>')+
```

Comment:
To me it looks like we should not match the `>` character in the name or fqn, but any character but the reserved ones (i.e. seperator and end character). Let me know if I am missing something

Reply:
yes, that's the goal of that pattern🙌

Reply:
it should not start with one or more `:`
it can have `>` in the middle but not at the end
it should not end with `:` or `>`

Reply:
the reason to put that there was that some entities like dashboards and pipelines that had `>` in the middle of the name which caused some issues if i remember correctly